### ISSUE TITLE
Request for update

### ISSUE THREAD

**Issue by Memory1014** [FINDarkside/TLD-Save-Editor-3#13]
Hello friend, I hope I won't bother you.
I am a Chinese. When I came into contact with this software for the first time, it successfully surprised me.
It really makes the game TLD more fun by at least 80%. But when I learned that you may not update this software again, it is really a pity, I believe many people think the same as me.
So if possible, we sincerely hope that you can slightly modify this software, just to make it read the archive of the current version of the TLD.
It's a really, really good mod and it's a shame it just disappeared.  Maybe you can start crowdfunding online, I believe there will be many people supporting you.
Finally, again a sincere request that you update this software, or just make it read the TLD's archives.
I hope this passage of mine didn't make you feel uncomfortable, and if it did, I sincerely apologize. Because I really need it too much, finally I wish you a happy life!

**Comment by FINDarkside** [comment]
Hi! Nice to hear you've enjoyed the editor. There's 2 people who have offered to help me in the development so it's likely we'll get something usable ready at some point. I can't really promise any timeline though. Also, one of the biggest reason for not wanting to maintain this project anymore is that since TLD is still in development it means that it will break again in the future and then I need to figure out what has changed.

> Finally, again a sincere request that you update this software, or just make it read the TLD's archives.

Out of curiosity what features would you want it to have to "read archives"? At the moment this save editor can already read the new saves (including inventory), but it still needs some work to be usable for normal users.

> I hope this passage of mine didn't make you feel uncomfortable, and if it did, I sincerely apologize. Because I really need it too much, finally I wish you a happy life!

No worries it's all good! I wish you a happy life as well! Let's hope we'll see this project released next year :blush:

**Comment by Memory1014** [comment]
Hello friends, it is a great honor to receive your reply.  At the same time, I am also sorry, because the major I studied cannot provide you with useful help, and I may not be able to provide you with some useful suggestions.
The editor version I used before is TLD Save Editor 2.18.1, this version is almost perfect in my opinion, it contains almost all the functions that players like me need, the most important thing is that it has  Chinese, this is so surprising to me.
It's just a pity that after the TDL game update, TLD Save Editor 2.18.1 cannot read the game save file, which means that the game save file cannot be edited.  So, if it is not very complicated, I think it is enough to make a slight modification on the basis of the source code of TLD Save Editor 2.18.1, so that it can read the game archive of the new version of TLD and edit it.  As for the items added in the new version of the game, it may not need to be added to the editor.
The above are some of my thoughts on this software, of course, in any case, I and other players will appreciate everything you and all developers have done, because you have made this game twice as fun.
Finally, I wish you good health and all the best!
